cmake/utils.cmake

```diff
@@ -150,15 +133,15 @@ function(add_ringmesh_utility bin_path)
     add_ringmesh_executable(${bin_path} "Utilities" ${ARGN})
     install(TARGETS ${exe_name} RUNTIME DESTINATION bin)
     set_target_properties(${exe_name} PROPERTIES 
-        RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin/utilities)
+        RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
 endfunction()
 
-function(add_ringmesh_test cpp_file_path)
+function(add_ringmesh_test cpp_file_path folder_name)
     add_ringmesh_executable(${cpp_file_path} "Tests" ${ARGN})
     # Add the test to CTest
     add_test(NAME ${exe_name} COMMAND ${exe_name})
     set_target_properties(${exe_name} PROPERTIES 
-        RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin/tests)
+        RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin/tests/${folder_name})
```

Comment:
Is it necessary?

Reply:
what do you mean? do you want to put test binaries in separate folder?